Momo's April Serenity updates (#931)
akeneo/pim-helpcenter: content/updates/2021-04/1-Review-your-proposals-more-efficiently.md

---
pim_announcement_alt_img: "New proposals screen"
pim_announcement_audience:
- EE
---

# Review your proposals more efficiently
::: meta-data type="Improvement" features="Productivity, Collaboration" available="late April" in="EE" link-to-doc="../articles/TODO"

We're glad to introduce the revamp of our proposals screen. It's now even more practical and efficient for those of you who, depending on their catalog rights, need to review the enrichment work of teammates. 

Among other improvements, you can notice that:

> - You now have, directly from within the proposal, a preview of your PDFs assets, your YouTube links, and your reference entity records. A great time-saver when reviewing them!

Reviewer: You forgot to mention the display of images (since the 4.0, we only displayed the asset code. The thumbnail was displayed before the 4.0 (PAM) but in black and white). We can now preview the media but also download them and copy their URL. And we display their label too.
For the records: this is the exact same display as in the PEF (label + thumbnail + link to the ref entity)

Author: Looks like I forgot a word. I reformulated to make it better. Is it better this way?

> - Proposals are now gathered by product. It will surely help you be more productive when validating or discarding proposals.
> - You also now have, directly from within the proposal, a preview of your PDFs assets, your YouTube links, and your reference entity records. A great time-saver when reviewing them!

Author: ```suggestion
- You now have, directly from within the proposal, a preview of your PDFs assets, your YouTube links, and your reference entity records. A great time-saver when reviewing them!
- Proposal layout is also now rearranged. Old and suggested new values are now easier to compare, making the whole process of values review more convenient.
```

---
pim_announcement_img: ./img/revamped-proposal-screen.png
pim_announcement_alt_img: "New proposals screen"
pim_announcement_audience:
- EE
---

> # Review your proposals more efficiently

Reviewer 2: efficiently or effectively @camillebourjade?

Reviewer 3: efficiently

::: meta-data type="Improvement" features="Productivity, Collaboration" available="late April" in="EE" link-to-doc="../articles/review-products-proposals.html"

> We're glad to introduce the revamp of our proposals screen. It's now even more practical and efficient for those of you who, depending on their catalog rights, need to review the enrichment work of teammates. 

Reviewer 2: We are pleased to present to you our new screen to manage proposals. This new screen has been thought to be more simple to use and straightforward and to make it easy and faster for those of you who, depending on your catalog rights, need to review your teammates or external contributors suggestions.

Author: I would better suggest this, as the screen is only useful for those who have the own rights on some categories.
```suggestion
We are pleased to present to you our new screen to manage proposals. This screen will make it easier and faster for those of you who, depending on your catalog rights, need to review your teammate or external contributor suggestions. Indeed, it has been thought to be more simple and straightforward to use. 
```

Reviewer 3: ```suggestion
We are happy to introduce our new screen to manage proposals. This screen will make it easier and faster for those of you who, depending on your catalog rights, need to review your teammate or external contributor suggestions. Indeed, it has been designed to be more simple and straightforward to use.
```

Among other improvements, you can notice that:

> - You now have, directly from within the proposal, a preview of your image assets, your PDFs, your YouTube links, and your reference entity records. A great time-saver when reviewing them!

Reviewer 2: ```suggestion
- You now have, directly from the proposal, a preview of your images, PDF files, YouTube links, and reference entity records. Being able to directly see and preview changes will surely help you save even more time!
```

> - You now have, directly from the proposal, a preview of your images, PDF files, YouTube links, and reference entity records. Being able to directly see and preview changes will surely help you save even more time!

Author: ```suggestion
- You now have, directly from the proposal, a preview of your images, PDF files, YouTube/Vimeo links, and reference entity records. Being able to directly see and preview changes will surely help you save even more time!
```

> - Proposal layout is also now rearranged. Old and suggested new values are now easier to compare, making the whole process of values review more convenient.

Reviewer 2: - The layout of the proposal has also been reorganized. Old and newly suggested values are now easier to compare, making the whole review process more convenient.

Author: Forgot to create a suggestion, Tata?
```suggestion
- The layout of the proposal has also been reorganized. Old and newly suggested values are now easier to compare, making the whole review process more convenient.
```

> - The layout of the proposal has also been reorganized. Old and newly suggested values are now easier to compare, making the whole review process more convenient.

Reviewer 3: ```suggestion
- The layout of the proposal has also been reorganized. Current and newly suggested values are now easier to compare, making the whole review process more practical.
```

> ![Reference entity records mass delete screen](../img/revamped-proposal-screen.png)

Author: ```suggestion
![New proposal screen](../img/revamped-proposal-screen.png)
```
